Sources/StreamChatSwiftUI/ChatChannelList/ChatChannelListItem.swift

```diff
@@ -270,14 +270,9 @@ public struct InjectedChannelInfo {
 extension ChatChannel {
 
     public var lastMessageText: String? {
-        if let latestMessage = latestMessages.first {
-            if let text = pollMessageText(for: latestMessage) {
-                return text
-            }
-            return "\(latestMessage.author.name ?? latestMessage.author.id): \(textContent(for: latestMessage))"
-        } else {
-            return nil
-        }
+        guard let latestMessage = latestMessages.first else { return nil }
+        let messageFormatter = MessagePreviewFormatter()
```

Comment:
we shouldn't create a new formatter for every item - let's put one in the `Utils`. You can also do `InjectedValues[\.utils]` in extensions.

Reply:
Since it is a struct and only contains logic, it is very lightweight, but if that option is available, I'll use it 👍

Reply:
Done ✅